config/locales/en.yml

```diff
@@ -373,3 +374,13 @@ en:
         one: Displaying <b>1</b> %{model}
         other: Displaying <b>all&nbsp;%{count}</b> %{model}
         zero: No %{model} found
+  webauthn_credentials:
+    incorrect_credentials: Your credentials are incorrect.
+    index:
+      go_settings: Register a WebAuthn credential
```

Comment:
Major websites are going with "security keys" (e.g. [Google](https://support.google.com/accounts/answer/6103523), [Facebook](https://www.facebook.com/help/401566786855239), [Twitter](https://help.twitter.com/en/managing-your-account/two-factor-authentication), [Shopify](https://help.shopify.com/en/manual/your-account/account-security/two-step-authentication#enable-two-step-authentication-with-security-keys), [GitHub](https://github.blog/2019-08-21-github-supports-webauthn-for-security-keys/)), naming things differently might be confusing for users who are not familiar with the underlying specifications.

Reply:
Agree it might be better than "WebAuthn credential".

Although still possibly confusing for a user to map the "Security Key" concept to "I used my Android Fingerprint" or "MacBook TouchID".

In general, "Security Key" might better match the idea of a roaming authenticator. Not so much a platform authenticator I guess. I wonder if for the latter, the "Device" concept might be more intuitive.

Reply:
Our original docs called it a 'security device' with similar underlying reasoning, we also tried to explain the difference between a USB key and a built-in biometric sensor. In the end we found these distinctions made it all just more confusing to read and we changed it to just 'security key' shortly thereafter.

Google is using the same naming everywhere, Chrome prompts me to choose "Use your security Key with Bluetooth/NFC/Fingerprint" on websites. I.e. a device can have a built-in security key or you can use an external one, but both are security keys.

Google also recently [announced that an Android phone's built-in security key can be used with a desktop or laptop](https://venturebeat.com/2019/04/10/you-can-now-use-your-android-phone-as-a-2fa-security-key-for-google-accounts/) (using the caBLE transport). So in technical terms; your phone's platform authenticator becomes also a roaming authenticator 🙈

Long story short, I'd suggest to not focus on the form factor or transport used but on the functionality: all of them are a security key with all the security properties you can expect from one 🙂

Reply:
Fair enough.